cmd/eve.go

```diff
@@ -255,7 +256,10 @@ var sshEveCmd = &cobra.Command{
 			if len(args) > 0 {
 				commandToRun = strings.Join(args, " ")
 			}
-			arguments := fmt.Sprintf("-o ConnectTimeout=5 -oStrictHostKeyChecking=no -i %s -p %d root@%s %s", eveSSHKey, eveSSHPort, getEVEIP(), commandToRun)
+			if eveHost == defaults.DefaultEVEHost {
```

Comment:
Seems, we can use `!cmd.Flags().Changed("eve-host")` here as more general.

Reply:
Done...